src/react-demo/Content/src/Hmr.fs

```diff
@@ -0,0 +1,71 @@
+module Hmr
+
+open Elmish
+open Elmish.Browser.Navigation
```

Comment:
please remove: Elmish.Browser.Navigation, Elmish.Browser.UrlParser and App.State